base/logging.c

```diff
@@ -626,6 +628,9 @@ gvm_log_func (const char *log_domain, GLogLevelFlags log_level,
                             message);
   g_free (prepend);
 
+  if (log_level == G_LOG_LEVEL_WARNING)
```

Comment:
I think this should include the more critical message types (which are represented by lower numbers): 
```suggestion
  if (log_level <= G_LOG_LEVEL_WARNING)
```